pkgs/misc/emulators/openmsx/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv, fetchFromGitHub, pkgconfig
+, python
+, alsaLib, glew, mesa_noglu, libpng
+, libogg, libtheora, libvorbis
+, SDL, SDL_image, SDL_ttf
+, freetype, tcl, zlib
+}:
+
+stdenv.mkDerivation rec {
+
+  name = "openmsx-${version}";
+  version = "git-2017-11-02";
+
+  src = fetchFromGitHub {
+    owner = "openMSX";
+    repo = "openMSX";
+    rev = "eeb74206ae347a3b17e9b99f91f2b4682c5db22c";
+    sha256 = "170amj7k6wjhwx6psbplqljvckvhxxbv3aw72jrdxl1fb8zlnq3s";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ pkgconfig python ];
+
+  buildInputs = [ alsaLib glew mesa_noglu libpng
+    libogg libtheora libvorbis freetype
+    SDL SDL_image SDL_ttf tcl zlib ];
+
+  customBuildFile = ./custom-nixos.mk;
+
+  postUnpack = ''
+    cp ${customBuildFile} $sourceRoot/build/custom.mk
```

Comment:
Semantically this fits better in `postPatch`, and this makes `$sourceRoot` unnecessary.  Also `customBuildFile` can be inlined:

```
postPatch = ''
  cp ${./custom-nixos.mk} build/custom.mk
'';
```